.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+on: push
+jobs:
+  job:
+    strategy:
+      matrix:
+        javaVersion: [1.8, 1.11]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Restore Coursier cache
+      uses: coursier/cache-action@v3
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{matrix.javaVersion}}
+    - name: Run tests
+      run: sbt ci
+      env:
+        JAVA_OPTS: "-XX:MaxMetaspaceSize=1g -Xms1g -Xmx2g -Xss2M"
+        SBT_OPTS: "-XX:+CMSClassUnloadingEnabled -XX:MaxMetaspaceSize=2g -Xmx2g"
```

Comment:
We're now up to 7G on the heap to run the tests :D

https://github.com/disneystreaming/weaver-test/blob/master/.drone.yml#L14